frontend/packages/helm-plugin/src/plugin.ts

```diff
@@ -1,3 +1,20 @@
-const plugin = [];
+import { FLAG_OPENSHIFT_HELM } from './const';
+import {
+  helmChartRepoModelProviderForBreadcrumbs,
+  helmChartRepositoriesBreadcrumbsProvider,
+} from './providers';
+
+const plugin = [
+  {
+    type: 'DetailPageBreadCrumbs',
+    properties: {
+      getModels: helmChartRepoModelProviderForBreadcrumbs,
+      breadcrumbsProvider: helmChartRepositoriesBreadcrumbsProvider,
+    },
+    flags: {
+      required: [FLAG_OPENSHIFT_HELM],
+    },
+  },
+];
```

Comment:
We should add new plugins to the `console-extensions.json` instead. Let me know if this doesn't work for some reason.